src/EnergyPlus/Psychrometrics.hh

```diff
@@ -1247,9 +1243,7 @@ namespace Psychrometrics {
 
 	inline
 	Real64
-	CPHW(
-		Real64 const Temperature // unused1208
-	)
+	CPHW()
```

Comment:
@mjwitte I'll look into this.  I'm open to the possibility that someday this could be dependent on temperature, but since that's far out of scope here, I'm not going to dig into that aspect.  My first instinct is, as you mentioned, just removing the function altogether, and just get a static value.  Actually, I'm fine with that.  We'll change it to a static value.  Later if we want to add a temperature dependent function, we should call it something else anyway, like CpfT(), and collapse the CPHW and CPCW to all call this one function.